R/load.project.R

```diff
@@ -275,7 +275,25 @@ load.project <- function(override.config = NULL)
   config <- .normalize.config(config,
                               setdiff(names(default.config), c("version", "libraries", "logging_level")),
                               .boolean.cfg)
-
+  
+  # add some project config if it exists in a file called custom.dcf
+  custom_cfg <- file.path('config', 'custom.dcf') 
+  if (file.exists(custom_cfg)) {
+        custom_cfg <- translate.dcf(custom_cfg)
+ 
+        # all items are char, so convert them if they look like logical or numeric
+        convert_type <- function(v){
```

Comment:
I'm wondering if there is a way to replace `convert_type` with `.boolean.cfg`. Even if there isn't, can you try to use the `.normalize.config` idiom above? It does exactly what you want to do here but in a reusable way!